web_poet/page_inputs/client.py

```diff
@@ -19,7 +19,7 @@
 _StrMapping = Dict[str, str]
 _Headers = Union[_StrMapping, HttpRequestHeaders]
 _Body = Union[bytes, HttpRequestBody]
-_Status = Union[str, int]
+_Status = Union[str, int, List[Union[str, int]]]
```

Comment:
What do you think about renaming it to _StatusList? It seems passing a list is what we should be promoting. Not sure why we support ints and strings in a first place :) List + a special case Literal("*") would be a bit cleaner (see https://docs.python.org/3/library/typing.html#typing.Literal). But that's out of scope for this PR.